moveit_core/controller_manager/include/moveit/controller_manager/controller_manager.h

```diff
@@ -120,12 +120,10 @@ class MoveItControllerHandle
     return name_;
   }
 
-  /** \brief Send a trajectory to the controller.
-   *
-   * The controller is expected to execute the trajectory, but this function call should not block.
-   * Blocking is achievable by calling waitForExecution().
-   * Return false when the controller cannot accept the trajectory. */
```

Comment:
regression in commenting

Reply:
@anasarrak can you please review this?

Reply:
https://github.com/ros-planning/moveit2/pull/6/commits/2e1eb3716fcd686e3b1ff515e5a87d3b3264b890